autoload/javacomplete.vim

```diff
@@ -206,6 +206,7 @@ augroup javacomplete
   autocmd BufEnter *.java,*.jsp call s:SetCurrentFileKey()
   autocmd BufWritePost *.java call s:RemoveCurrentFromCache()
   autocmd VimLeave * call javacomplete#server#Terminate()
+	autocmd VimLeavePre * call javacomplete#server#Stop()
```

Comment:
Tabs vs. Spaces debate will never end. Newest commit should match the (crazy un-customizable) spaces styling